flocker/node/agents/blockdevice.py

```diff
@@ -883,6 +883,8 @@ def destroy_volume(blockdevice_id):
 
         :raises UnknownVolume: If the supplied ``blockdevice_id`` does not
             exist.
+            Note that the ebs and cinder implementation can raise a
```

Comment:
I'm not sure this should be on interface documentation.

In general in Python (and really most languages with exceptions AFAIK, even Java with supposedly explicit exceptions) you have to assume that there are additional unexpected random exceptions that will get thrown for some error cases. It's worth documenting this in the relevant implementation docstrings, though.